src/components/SourceAppConfigs.tsx

```diff
@@ -1,18 +1,18 @@
 import { ThemeOptions, } from '@mui/material'
-import { adKnowledgePortal, adKnowledgePortalStaging } from 'configs/adknowledgeportal'
```

Comment:
Clean up app configurations.  No longer need to have a staging version, since apps will provide their own redirect URLs.  Will will hard code the production app URL for the upcoming known Sage apps page, though.